source/features/deep-reblame.tsx

```diff
@@ -47,7 +47,7 @@ const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number):
 
 async function redirectToBlameCommit(event: DelegateEvent<MouseEvent, HTMLAnchorElement>): Promise<void> {
 	const blameLink = event.delegateTarget;
-	if (blameLink.href !== '#deep-reblame') {
+	if (blameLink.href) {
```

Comment:
```suggestion
	if (blameButton instanceof HTMLButtonElement) {
```

Reply:
I need the other way around (Not a button element) What is that called (an A element)